### PR TITLE
Webflow edit page

### PR DESCRIPTION
**Pull request by FoamyGuy**
replaces: #6592 

I tried getting the branch from that PR straightened out, but could not work it out. I started this one from an up-to-date main and didn't run into the same trouble with submodules on it. 

One additional tweak in this version is specifying the quote_characters for jsmin in order to get it handle ES6 strings with backticks better. The `edit.js` file makes use of this string syntax. https://pypi.org/project/jsmin/ mentions that it does not intend to support ES6, but does provide this argument if backtick strings are the specific thing causing trouble. I retested the edit page after this and it's back to working correctly to load and save files on the drive.

### CODE REVIEW

File: supervisor/shared/web_workflow/static/edit.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.6.0/ace.min.js"
+            integrity="sha512-Ky7AOm/5oRYp5QzV9diL95tE/OKjzfAkugQ+llHy1scOlzIyAt2SoyriapPAZTvtZNL/xbYI1Gvt5jJYurPBdw=="
+            crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.6.0/mode-python.min.js"
+            integrity="sha512-SdtfSOaR+TnSvGsZ2dmErFrcMC/CK6J/l2kNaXv3AU9BtNzqLDtK69ImVp6zOAY9Udii9GrtH7NssygOh/w0hg=="
+            crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.6.0/ext-settings_menu.min.js"
+            integrity="sha512-GqA/hV/4FrUn8lUmY+5EWvrB4Bbw3iv0TMY4wVFLWp+OyMqhKhYWXCtYA0+X68TdEA8nUq3og+d6VLTtwprvyw=="
+            crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```
Comment (tannewt) [inline supervisor/shared/web_workflow/static/edit.html:28]: I'd rather not have any extra deps that require the browser to have general internet access. I want to link to code.circuitpython.org for a full editing experience.

**Review by tannewt — CHANGES_REQUESTED** [review]
Do you have a version that is textarea + js without deps? I was hoping it'd work offline. code.circuitpython.org can be used when online.

File: supervisor/shared/web_workflow/static/edit.html (hunk 1 above)
Comment (FoamyGuy) [reply supervisor/shared/web_workflow/static/edit.html]: The latest commits change over to a more basic version with just <textarea> interface. Also merged main and resolved conflicts.

**Comment by tannewt** [comment]
#6608 includes a fix for the build that runs out of space.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**